### PR TITLE
Task/railsconf 29 update t in index.html.erb

### LINKED ISSUE #3463: Railsconf 29: update t() in app/views/case_assignments/index.html.erb
Thank you for coming to our workshop!

To complete this issue:
1. open `app/views/case_assignments/index.html.erb`
1. find each call of `t(STRING)`
1. look up `STRING` in `config/locales/views.en.yml`
    1. the YAML file is nested, first with the language (always `en` here), then the view name, then the action name
    1. so for your `STRING`, look under `en:` then `index.html.erb:` then `:` and then `STRING:`
    1. if you need any help figuring the view or action name to look up the translation with, please ask a workshop helper
1. replace the usage of `t(` with the translation you found in `config/locales/views.en.yml`

This will simplify our templates and make it easier to maintain the code going forward.

Example:

```
Before:
<span><%= t(".title") %></span>

In config/locales/views.en.yml:
en:
  index.html.erb:
    :
      title: Hello world

After:
<span>Hello world</span>
```

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by ErinClaudio**
This is a draft I have failing test

### What github issue is this PR for, if any?
Resolves #3463

### What changed, and why?

- Removed the t("") from the file and replaced it with text found in yml file. 
- This Translation feature is no longer being used.

### How will this affect user permissions?
- Volunteer permissions: N/A
- Supervisor permissions: N/A
- Admin permissions: N/A

### How is this tested? (please write tests!) 💖💪
Not tested

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

File: app/views/case_assignments/index.html.erb [hunk 1]
```diff
@@ -1,29 +1,29 @@
-<h1><%= "#{t(".title")} #{@volunteer.display_name}" %></h1>
+<h1><%= "#{Editing} #{@volunteer.display_name}" %></h1>
```
Comment (compwron) [inline app/views/case_assignments/index.html.erb:1]: syntax issue

Comment (ErinClaudio) [reply app/views/case_assignments/index.html.erb:1]: @compwron GitHub states that this is merged. Am I correct in that I do not need to make a change to line 1?

Comment (compwron) [reply app/views/case_assignments/index.html.erb:1]: This is merged but the next step is to add a test to this PR https://github.com/rubyforgood/casa/pull/3552
Can you take on adding a test to this pr? :) https://github.com/rubyforgood/casa/pull/3552

Comment (ErinClaudio) [reply app/views/case_assignments/index.html.erb:1]: A couple of questions

- This issue is in the index view of the case_assignments_controller? I do not see an index method within that controller? Did I miss the method?

- There are no tests for the case_assignments_controller. Should I create those files and start writing tests for the view file?

**Pull request merged**